src/js/hca/components/form-elements/ErrorableSelect.jsx

```diff
@@ -0,0 +1,131 @@
+import React from 'react';
```

Comment:
Maybe in a future PR we can move these form-element or question components to a directory outside of HCA so they can be used by other apps, but for now this is good :)